README.md

```diff
@@ -95,15 +95,27 @@ For the nRF52840 you'll want to use the [`nrf52840-hal`].
 
 #### 5. Import your HAL
 
-Now that you have selected a HAL fix the HAL import in `src/lib.rs`
+Now that you have selected a HAL, fix the HAL import in `src/lib.rs`
 
 ``` diff
  // my-app/src/lib.rs
 -// use some_hal as _; // memory layout
 +use nrf52840_hal as _; // memory layout
 ```
 
-#### 6. run!
+#### 6. Get a linker script
+
+Some HAL crates require that you copy over a file called `memory.x` from the HAL to the root of your project. For nrf52840-hal, the file is in the folder `nrf-hal/nrf52840-hal/`. You can get it from your local Cargo folder, the default location is:
```

Comment:
The nRF HAL does not require any work on the user's side

Reply:
Aah my bad, never worked with the nrf52840-hal but figured I'd keep that as the example. I'll make an edit!